### PR TITLE
added event editing

### PR DESCRIPTION
**Pull request by ShemarG**

### CODE REVIEW

File: client/src/components/EditEvent.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+                res.json().then(json => {
+                    setTitle(json.title)
+                    setVirtual(json.isvirtual)
+                    setLocation(json.location)
+                    setDescription(json.description)
+                    set_time_start(json.time_start)
+                    set_time_end(json.time_end)
+                })
+            } else {
+                console.log('there was an error')
```
Comment (Ricardo-21) [inline client/src/components/EditEvent.js:43]: Maybe throw an error instead?

**Review by Ricardo-21 — CHANGES_REQUESTED** [review]
All looks good! Just the console logs that have to get removed feel free to merge in!

File: server/controllers/eventsController.js [hunk 2]
```diff
@@ -84,22 +84,21 @@ const deleteEvent = async (req, res) => {
 
 const editEvent = async (req, res) => {
     const event_id = req.params.id;
-    const user_id = '1';//Temporary until session cookie is available
-    const updateEvent = {title: 'New title', isVirtual: false};
-
-    const event = await Event.getEvent(event_id);
-
+    console.log(req.body)
```
[the remark this answers is not in the document]
Comment (ShemarG) [reply server/controllers/eventsController.js:87]: Good catch!

File: client/src/components/EditEvent.js (hunk 1 above)
Comment (ShemarG) [reply client/src/components/EditEvent.js:43]: Planned to do another PR just for errors down the road, for now they're just a bandaid, but a necessary bandaid

File: server/controllers/eventsController.js [hunk 3]
```diff
@@ -84,22 +84,21 @@ const deleteEvent = async (req, res) => {
...
+            res.status(200).json(updatedEvent)
         }
         else {
             throw new Error ('users do not match')
         }
     }
-    catch {
-
+    catch (err) {
+        console.log(err)
```
[the remark this answers is not in the document]
Comment (ShemarG) [reply server/controllers/eventsController.js]: In the errors PR I'm going to pull in my custom error handling middleware for this, back burner until then.

**Pull request merged**